src/main/java/edu/ucsb/cs56/ucsb_courses_search/controller/CSVDownloadController.java

```diff
@@ -10,22 +14,39 @@
 import edu.ucsb.cs56.ucsbapi.academics.curriculums.v1.classes.CoursePage;
 import edu.ucsb.cs56.ucsbapi.academics.curriculums.v1.classes.Course;
 
+import edu.ucsb.cs56.ucsb_courses_search.repository.ScheduleItemRepository;
+import edu.ucsb.cs56.ucsb_courses_search.entity.ScheduleItem;
+
+import java.util.ArrayList;
+
 import java.io.IOException;
 
 import javax.servlet.http.HttpServletResponse;
 
 import java.util.ArrayList;
 import java.util.List;
 
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
 import org.springframework.web.bind.annotation.RestController;
 
 @RestController
 public class CSVDownloadController {
 
+    private Logger logger = LoggerFactory.getLogger(CourseController.class);
 
     @Autowired
     private CurriculumService curriculumService;
 
+    @Autowired
+    private ScheduleItemRepository scheduleItemRepository;
+
+    @Autowired
+    public CSVDownloadController(ScheduleItemRepository scheduleItemRepository) {
+        this.scheduleItemRepository = scheduleItemRepository;
```

Comment:
Is this constructor necessary?

Reply:
Not sure why Jerry included it but I'll mess around with it and see if it's needed